stages/test/test_mkfs_ext4.py

```diff
@@ -82,6 +82,10 @@ def test_mkfs_ext4_integration(tmp_path):
     ({}, []),
     ({"verity": True}, ["-O", "verity"]),
     ({"verity": False}, ["-O", "^verity"]),
+    ({"orphan_file": True}, ["-O", "orphan_file"]),
+    ({"orphan_file": False}, ["-O", "^orphan_file"]),
+    ({"metadata_csum_seed": True}, ["-O", "metadata_csum_seed"]),
+    ({"metadata_csum_seed": False}, ["-O", "^metadata_csum_seed"]),
```

Comment:
Thanks, bonus points for having one test that adds all three together (this would ensure we never accidentially make the selection of the feature an if/elif/elif/else construct which is super unlikely but but might still be worth the one extra line here.

Reply:
Sure, I can probably add that in!